.npmrc

```diff
@@ -0,0 +1,3 @@
+@pmedianetwork:registry=https://adverity-508912190628.d.codeartifact.eu-west-1.amazonaws.com/npm/adverity-repo/
+//adverity-508912190628.d.codeartifact.eu-west-1.amazonaws.com/npm/adverity-repo/:always-auth=true
+//adverity-508912190628.d.codeartifact.eu-west-1.amazonaws.com/npm/adverity-repo/:_authToken=${CODEARTIFACT_AUTH_TOKEN}
```

Comment:
how will this work for a developer working in this repo and having this in such form?

Reply:
the develop must execute

```
export CODEARTIFACT_AUTH_TOKEN=`aws codeartifact get-authorization-token --domain adverity --domain-owner 508912190628 --query authorizationToken --output text`
```

before he can do npm install

Reply:
I'm not sure if this can be automized. Since it also depends on the default aws profile of the dev

Reply:
yeah so the cli setup is one, but weren't we talking about some npm pre hook to automize that?

Reply:
yeah.. that's what I'm actually concerned of

so in my case I have to do :

```
AWS_PROFILE=adverity-repo aws codeartifact get-authorization-token
```

But that's just because in my `./aws/credentals` it's called `[adverity-repo]`

but this might not be the case for every developer

Reply:
So if there's one profile it should be default and there's not problem. 
But if the user has more then we can require it has a certain name? e.g. `repo`
Then we could do `AWS_PROFILE="$(aws configure list-profiles | grep repo)" aws codeartifact get-authorization-token....` ? this might standing in your line sometimes....

and have that in npm `preinstall` script?

Reply:
`aws codeartifact get-authorization-token.... --profile "$(aws configure list-profiles | grep repo)"`

Reply:
I did some testing.. I this doesnt work.. because of the `nvm` issue I told you in stand up today.

But I came to the conlusion.. the best way for developers is to actually put the export into `.bashr`, `.zshrc` or whatever

I'm gonna rename the exported variable to `ADVERITY_CODEARTIFACT_AUTH_TOKEN` so it does not potentially conflict

Reply:
https://github.com/pmedianetwork/eslint-config/pull/87/commits/d148974ce81174f3eaf495003493d0b0b2677799

Reply:
ok